torch/_torch_docs.py

```diff
@@ -4338,6 +4338,80 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(self, other, *, out=None) -> Tensor
+
+Compute ``self * log(other)`` so that ``0 * log(0)`` returns zero.
```

Comment:
"Compute" -> "Computes"

"so that" -> "except treats \`\`0 * log(0)\`\` as zero."